scripts/start-new-version.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
 # Note: Run this from within the root directory
 
 version=$1
 
-./build.sh -s "version.cake" -t "StartNewVersion" -NewVersion="$version"
+./build.sh -s "version.cake" --t="StartNewVersion" --NewVersion="$version"
```

Comment:
Is this line tested? That looks like a regular bash named arguments passing, and those are commonly used with single hyphen (as it was previously: `...-t "StartNewVersion"`).

Reply:
Yes, it was tested.

Reply:
I use `./build.ps1 -target Externals` format...

Reply:
Reworked